.github/workflows/tests.yml

```diff
@@ -60,6 +60,9 @@ jobs:
       - name: Run unit tests
         run: yarn test
 
+      - name: Check translation files
+        run: yarn gen-translations --fail-on-update
```

Comment:
Would be nice to make this give particular instructions if it fails (e.g. explain what this failure means and how to fix it)